core/modules/qproc/testIndexMap.cc

```diff
@@ -86,10 +86,25 @@ BOOST_FIXTURE_TEST_SUITE(Suite, Fixture)
 
 BOOST_AUTO_TEST_CASE(SecLookup) {
     ConstraintVector cv;
-    char const* argv[3] = {"111", "112","113"};
-    cv.push_back(makeConstraint("sIndex", 3, argv));
+    short const size = 3;
+    char const* argv[size] = {"111", "112","113"};
+    cv.push_back(makeConstraint("sIndex", size, argv));
 
     ChunkSpecVector csv = si.lookup(cv);
+
+    std::cout << "SecLookup\n";
+    std::copy(csv.begin(), csv.end(),
+              std::ostream_iterator<ChunkSpec>(std::cout, ",\n"));
+}
+
+BOOST_AUTO_TEST_CASE(SecLookupMultipleObjectId) {
+    ConstraintVector cv;
+    short const size=5;
```

Comment:
Use standard `int` type for everything unless you are concerned with storage size or platform-specific issues.